Cargo.toml

```
@@ -17,7 +17,8 @@ tokio-postgres = { version = "^0.7.7", features = ["with-serde_json-1"] }
 env_logger = "0.9"
 log = "0.4"
 http = "0.2"
-tonic = {version = "0.8", features = ["tls", "tls-roots"]} # Use system trust roots.
+tonic = { version = "0.10", features = ["tls", "tls-roots"] } # Use system trust roots.
 serde = { version = "1", features = ["derive"] }
 futures = "0.3"
 async-trait = "0.1.68"
+serde_json = "1.0.127"
```

We usually avoid specifying patch version

```suggestion
serde_json = "1.0"
```

---

I generally [don't think that's a good idea](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277), especially considering that serde_json and other dtolnay crates don't follow semantic versioning.